config/locales/es.yml

```diff
@@ -1448,15 +1448,15 @@ es:
     reimbursement_mailer:
       reimbursement_email:
         days_to_send: Días para envío
-        dear_customer: Estimado comprador
+        dear_customer: Estimado Cliente,
         exchange_summary: Resumen de intercambio
         for: Para
-        instructions: Instrucciones
+        instructions: Se ha procesado su reembolso.
         refund_summary: Resumen de reembolso
         subject: sujeto
         store_team: 'Equipo de %{store_name}'
-        total_refunded: Total de reembolso
-        thanks: Gracias por tu compra.
+        total_refunded: Total de reembolso:
```

Comment:
Psych::SyntaxError: (/home/travis/build/spree-contrib/spree_i18n/lib/spree_i18n/../../config/locales/es.yml): mapping values are not allowed in this context at line 1458 column 43

to remove ":"